cmd/ipfs/daemon.go

```diff
@@ -274,6 +274,7 @@ func daemonFunc(req cmds.Request, res cmds.Response) {
 		}
 	}
 
+	fmt.Printf("Daemon is ready\n")
```

Comment:
as long as it's the last thing printed, it can be ok. servers dont have to be running. maybe we can do this with a wait group? (add N to the wait group, and do `wg.Done()` as the lines are printed. the last one is:

```go
go func () {
  wg.Wait()
  fmt.Println("Daemon is ready!")
}()
```